docs/design/in-doc-search-ui.rst

```diff
@@ -0,0 +1,183 @@
+In Doc Search UI
+================
+
+Giving readers the ability to easily search the information
+that they are looking for is important for us.
+We have already upgraded to the latest version of `Elasticsearch`_ and
+we plan to implement `search as you type` feature for all the documentations hosted by us.
+It will be designed to provide instant results as soon as the user starts
+typing in the search bar with a clean and minimal frontend.
+This design document aims to provides the details of it.
+This is a GSoC'19 project.
+
+.. warning::
+
+    This design document details future features that are **not yet implemented**.
+    To discuss this document, please get in touch in the `issue tracker`_.
+
+
+The final result may look something like this:
+
+.. figure:: ../_static/images/design-docs/in-doc-search-ui/in-doc-search-ui-demo.gif
+    :align: center
+    :target: ../_static/images/design-docs/in-doc-search-ui/in-doc-search-ui-demo.gif
```

Comment:
I think if you put just `in-doc-search-ui-demo.gif` in both places here, Sphinx will just do the magic. Not sure, though

Reply:
I just tried it.
if we do this... then it is looking for the image in the same folder i.e., `design/in-doc-search-ui.gif`.